libs/shared/src/lib/components/provider/index.tsx

```diff
@@ -13,6 +14,7 @@ export function BytebankProvider({ canNavigate = true, children }: { routes?: Pr
                 <UserProvider>
                     {canNavigate ? <BytebankHeader /> : ''}
                     <Box display={"flex"} minHeight="100vh"> {children} </Box>
+                    <BytebankFooter />
```

Comment:
A página de erro preferencialmente não tem navegação.

```suggestion
                   {canNavigate ?  <BytebankFooter /> : ''}
```